encoding/masking-limiting-etc.md

```diff
@@ -33,8 +52,47 @@ and one of these formats:
 `GRAY16` (16 bits integer),
 or `GRAYS` (single precision floating point).
 
-Vapoursynth includes some basic tools for manipulating masks as well:
+#### [std.MaskedMerge][]
+
+This is the main function for masking
+that performs the actual merging.
+It takes three clips as input:
+clipa, clipb and one mask clip.
+The output will be a convex combination of the input clips,
+where the weights are given by the brightness of the mask clip,
+like this
+(the variables refer to the value of the same pixel in each clip):
+
+```py
+output = clipa * (MAX_VALUE - mask) + clipb * mask # in 8 bpp for example, MAX_VALUE would be 255
+```
+
+In simpler terms:
+for brighter areas in the mask,
+the output will come from clipb,
```

Comment:
I suggest using inline code blocks when referring to variable names, so `` `clipb` ``. (Actually I went ahead and did that.)